### PR TITLE
Added a folder for Responsive_Design_and_Media_Queries

### PR DESCRIPTION
**Pull request by chaudharydivya199**
I have added content for Responsive design and Media Queries #1029

### CODE REVIEW

File: Web_Development/FrontEnd/Responsive_Design_and_Media_Queries/Responsive_Design_and_Media_Queries.md [hunk 1]
```diff
@@ -0,0 +1,135 @@
+
+<p align="center">
+<img src="https://miro.medium.com/max/4800/1*YoXg1v9QIVbSKNaTIzr38A.jpeg" width="60%" height="60%" >
+</p>
+
+<h1 align="center">Responsive Design & Media Queries</h1>
+
+<p>&nbsp;</p>
```
Comment (praveenscience) [inline Web_Development/FrontEnd/Responsive_Design_and_Media_Queries/Responsive_Design_and_Media_Queries.md:8]: Please get rid of all the HTML. Use only Markdown.

File: Web_Development/FrontEnd/Responsive_Design_and_Media_Queries/Responsive_Design_and_Media_Queries.md [hunk 2]
```diff
@@ -0,0 +1,135 @@
...
+
+# **What is responsive design?**
```
Comment (praveenscience) [inline Web_Development/FrontEnd/Responsive_Design_and_Media_Queries/Responsive_Design_and_Media_Queries.md:10]: No need to bold already bold H1.

**Review by praveenscience — CHANGES_REQUESTED** [review]
Please go through all the changes and make it.

**Comment by chaudharydivya199** [comment]
@praveenscience  please review my PR

**Comment by praveenscience** [comment]
@chaudharydivya199 Can you please use the right PR template?

**Comment by chaudharydivya199** [comment]
Hello sir but sir I made all the changes .I removed html syntax

**Comment by chaudharydivya199** [comment]
@praveenscience  Sir can you please tell me how to use right template PR After that I will make changes. Sir i am confused how to make

**Pull request closed without merging**